ui/src/header/styles.scss

```diff
@@ -1,15 +1,15 @@
 header {
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
 
   .header-item {
     display: flex;
     justify-content: space-around;
     align-items: center;
   }
 
-  .header-galaxy-logo {
-    max-height: 6em;
+  .header-make-a-wish-logo {
+    padding-top: 40px;
```

Comment:
The padding makes the first screen way too close to the text input fields. 

<img width="942" alt="Screen Shot 2019-10-18 at 11 23 54 AM" src="https://user-images.githubusercontent.com/6162814/67107107-cd68d280-f199-11e9-89ba-c93d5ae3ae18.png">

The first screen has the SplashScreen div adding 2.5em in padding all the way around and it gets changed to `root` when we start working through the data entry portion. We probably want to make that consistent through the workflow so that the logo position stays consistent as well.

Reply:
Done